### ISSUE TITLE
Notifications no longer showing up;  Hover labels broken

### ISSUE THREAD

**Issue by Balentay** [typefully/minimal-twitter#193]
I think the latest update broke something because the bell isn't showing notifications.  They appear in the tab but not on the site.

Additionally hover labels seem to be broken.  After refreshing the page I need to disable and then re-enable them to get them to work properly

**Comment by deadcoder0904** [comment]
Yes, this too as well as view count is not showing up. Something broke.

**Comment by Dineshs91** [comment]
These issues are fixed in the latest version of the extension.
